safe_transaction_service/history/views.py

```diff
@@ -66,13 +66,20 @@ class AboutView(APIView):
 
     renderer_classes = (JSONRenderer,)
 
-    @method_decorator(cache_page(60 * 60))  # 1 hour
+    # @method_decorator(cache_page(60 * 60))  # 1 hour
```

Comment:
I only saw this now but was this intended? @Uxio0

Reply:
No. I disabled cache to test the headers

Reply:
Thanks for noticing

Reply:
https://github.com/gnosis/safe-transaction-service/pull/757